packaging/torchvision/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - libpng
     - ffmpeg >=4.2  # [not win]
```

Comment:
What about ffmpeg?

Reply:
This one is consistent across all config and that's why I left the pinning on. Happy to unpin it but the unit-tests are still broken and we won't be able to tell if we mess up something...

@seemethere Any thoughts on whether this can be unpinned?

Reply:
Okay, what about pyav?

Reply:
Yes on master in some places it's pinned and in some places it's not. See:

https://github.com/pytorch/vision/blob/851b623c8c7236dc6c5d7e0ec904a2576a965ff7/.circleci/unittest/linux/scripts/environment.yml#L19
https://github.com/pytorch/vision/blob/851b623c8c7236dc6c5d7e0ec904a2576a965ff7/packaging/torchvision/meta.yaml#L55

Reply:
I see, makes sense.